pages/k8s/tigera-secure-ee.md

```diff
@@ -40,11 +37,10 @@ Before you start, you will need:
   </div>
 </div>
 
-
-To start, deploy **Charmed Kubernetes** with Tigera Secure EE:
+Deploy the `charmed-kubernetes` bundle with the [tigera overlay][tigera-overlay]:
 
 ```bash
-juju deploy containers-kubernetes-tigera-secure-ee
+juju deploy charmed-kubernetes --overlay flannel-overlay.yaml
```

Comment:
this is a trick, right? you mean tigera-overlay.yaml

Reply:
yikes.  fixed.